### DISCUSSION TITLE
variable raised to power of another variable gives an error

### DISCUSSION THREAD

**Discussion by notesofdabbler** [infiniteopt/InfiniteOpt.jl#211] [Q&A] [answered]
I am trying a small example of fitting both rate constant and reaction order. So the ODE is of the form 
`dc/dt = -kc^n`. I was trying to code as shown below and it gives an error where I define the derivative. 
<img width="1103" alt="Screen Shot 2022-01-26 at 11 54 22 PM" src="https://user-images.githubusercontent.com/7705554/151294412-6b28e517-0a24-4fbe-8b0f-96fcfeedf0dc.png">
<img width="1022" alt="Screen Shot 2022-01-26 at 11 54 39 PM" src="https://user-images.githubusercontent.com/7705554/151294425-c883cb0c-e264-4953-bedf-6d809245e9ad.png">

**Comment by pulsipher** [comment, marked as answer]
I am unable to replicate the error on the current version `v0.5.2`. Which version of InfiniteOpt are you running? You can check via the `status` command in the package manager. General nonlinear expressions like this one require at least version `v0.5`.

**Comment by notesofdabbler** [comment]
Thanks. Updating the version fixed it. For some reason, even when I tried update InfiniteOpt it was not updating the version and then I had to install it using the specific version command add InfiniteOpt@0.5.2. Once I did it, the above code worked.
